### PR TITLE
Refactor /highlight

### PR DESCRIPTION
**Pull request by KrisXV**
This removes the need for a comma after `add` and `delete` in `/hl add/delete` and makes error messages display as red.

### CODE REVIEW

**Comment by KrisXV** [comment]
bump @Zarel

File: js/client-chat.js [hunk 1]
```diff
@@ -1115,13 +1124,15 @@
 				case 'highlight':
 				case 'hl':
 					this.add('Set up highlights:');
-					this.add('/highlight add, [word] - Add the word [word] to the highlight list.');
-					this.add('/highlight roomadd, [word] - Add the word [word] to the highlight list of whichever room you used the command in.');
+					this.add('/highlight add [word 1, word 2, ...] - Add the provided list of words to your highlight list.');
```
Comment (Zarel) [inline js/client-chat.js:1127]:
```suggestion
					this.add('/highlight add [word 1], [word 2], [...] - Add the provided list of words to your highlight list.');
```

I think this makes it clearer that you're actually supposed to type the comma if you have multiple words.

File: js/client-chat.js [hunk 2]
```diff
@@ -792,37 +792,39 @@
 			case 'highlight':
 				if (this.checkBroadcast(cmd, text)) return false;
 				var highlights = Dex.prefs('highlights') || {};
-				if (target.indexOf(',') > -1) {
-					var targets = target.match(/([^,]+?({\d*,\d*})?)+/g);
+				if (target.indexOf(' ') > -1) {
```
Comment (Zarel) [inline js/client-chat.js:795]:
```suggestion
				if (target.includes(' ')) {
```

**Review by Zarel — APPROVED** [review]
Looks good other than my two suggestions.

**Comment by Zarel** [comment]
(Although note that my second suggestion also applies to three other lines I didn't explicitly suggest on.)

**Comment by KrisXV** [comment]
did @Zarel

**Comment by Zarel** [comment]
Thanks!

**Pull request merged**